.github/workflows/test.yml

```
@@ -0,0 +1,53 @@
+name: test CI
+
+on: [pull_request]
+
+jobs:
+  pytest_with_coverage:
+    runs-on: ubuntu-latest
```

I'd like specify the specific version
```suggestion
    runs-on: ubuntu-20.04
```